<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="3" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="31609" started-at="2020-09-30T18:44:32Z" finished-at="2020-09-30T18:45:04Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="31609" started-at="2020-09-30T18:44:32Z" finished-at="2020-09-30T18:45:04Z">
      <class name="ecommerce.e_shop">
        <test-method status="PASS" signature="browserlaunch()[pri:0, instance:ecommerce.e_shop@62ee68d8]" name="browserlaunch" duration-ms="13124" started-at="2020-09-30T18:44:32Z" finished-at="2020-09-30T18:44:45Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- browserlaunch -->
        <test-method status="PASS" signature="NewArrivals()[pri:1, instance:ecommerce.e_shop@62ee68d8]" name="NewArrivals" duration-ms="18439" started-at="2020-09-30T18:44:46Z" finished-at="2020-09-30T18:45:04Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- NewArrivals -->
        <test-method status="FAIL" signature="selectItemsToCart()[pri:2, instance:ecommerce.e_shop@62ee68d8]" name="selectItemsToCart" duration-ms="2" started-at="2020-09-30T18:45:04Z" finished-at="2020-09-30T18:45:04Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at ecommerce.e_shop.selectItemsToCart(e_shop.java:74)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- selectItemsToCart -->
      </class> <!-- ecommerce.e_shop -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
